### ISSUE TITLE
Poverty and inequality measures should accept more than one "group_by" column

### ISSUE THREAD

**Issue by rafapereirabr** [ipeaGIT/accessibility#36]
Poverty and inequality measures should accept more than one "group_by" column.

## reprex
```
library(accessibility)

# load data
data_dir <- system.file("extdata", package = "accessibility")
travel_matrix <- readRDS(file.path(data_dir, "travel_matrix.rds"))
land_use_data <- readRDS(file.path(data_dir, "land_use_data.rds"))

access <- cumulative_cutoff(
  travel_matrix,
  land_use_data,
  cutoff = 30,
  opportunity = "jobs",
  travel_cost = "travel_time"
)

# create scenarios
landuse1 <- landuse2 <- copy(land_use_data)
landuse1$scenario <- 'before'
landuse2$scenario <- 'after'

# create race groups
landuse1$race <- landuse2$race <- sample(c("W","B"), size = nrow(landuse1), replace = T)

new_land_use_data <- rbind(landuse1, landuse2)

gini_index(
  accessibility_data = access,
  sociodemographic_data = new_land_use_data,
  opportunity = "jobs",
  population = "population", 
  group_by = c("scenario", "race")
)
```
> Error in assert_accessibility_data(accessibility_data, opportunity, group_by) : Assertion on 'accessibility_data' failed: Names must include the elements {'id','jobs','scenario','race'}, but is missing elements {'scenario','race'}.

**Comment by rafapereirabr** [comment]
ooops. There was an error in my code.

This should work fine:

```
# create race groups
landuse1$race <- landuse2$race <- sample(c("W","B"), size = nrow(landuse1), replace = T)

new_land_use_data <- rbind(landuse1, landuse2)

new_access <- merge(access, 
                    new_land_use_data[, .(id, scenario, race )], 
                    by = 'id',
                    all = T)

accessibility::gini_index(
  accessibility_data = new_access,
  sociodemographic_data = new_land_use_data,
  opportunity = "jobs",
  population = "population", 
  group_by = c("race", "scenario")
)
```

closing this issue

**Comment by dhersz** [comment]
Actually, they should be present only in `accessibility_data`.
